app/models/bundle_context.rb

```diff
@@ -1,6 +1,9 @@
+require 'fileutils'
```

Comment:
This is unnecessary to use `FileUtils` here.  Compare to `DigitalObject`.

Because of the way rails auto-loading works, if it were required, it is more typical to put the `require` in a "central" location, so all the app has access to it.

Reply:
I was getting an error when using FileUtils.mkdir_p, and adding this require fixed it.  I don't think anyplace else in the app will need it, so I figured here was a good spot for the require.

Reply:
#317

Reply:
That doesn't make sense to me since DigitalObject also uses `FileUtils` just fine and does not need a similar `require` statement.